logger/logger.go

```diff
@@ -44,3 +36,12 @@ func Get() zerolog.Logger {
 
 	return log
 }
+
+func SetLogLevel(logLevel zerolog.Level) zerolog.Logger {
+	lock.Lock()
```

Comment:
[q] why is a lock needed here? Would this prevent concurrent runs of `PersistentPreRun` or force them to be executed serially?

Reply:
Eh this is really probably poorly done. The idea is not so much guarding against `PersistentPreRun`, but more against usage of this as a library. Since it's updating the module-level variable, I want to be sure there wasn't any conflicting... anything. But maybe this is just overkill. I can pull it out for now until we have a solid need for it.